accelerometer/device.py

```diff
@@ -9,6 +9,54 @@
 import struct
 from subprocess import call
 import sys
+import jpype
+import atexit
+import shutil
+from tempfile import mkstemp
+import re
+
+
+
+def parse(inputFile, outputFile, **kwargs):
+
+    if inputFile.endswith('.gz'):  # Decompress to temporary file
+        ext = re.search(r'(\.\w+)\.gz', inputFile).group(1)
+
+        _inputFileFd, _inputFile = mkstemp(suffix=ext)
+
+        with open(_inputFile, 'wb') as _:
+            shutil.copyfileobj(gzip.open(inputFile), _)
+
+        inputFile = _inputFile
+
+    # Start JVM
+    if not jpype.isJVMStarted():
+        jpype.addClassPath('../java/')
+        jpype.addClassPath('../java/JTransforms-3.1-with-dependencies.jar')
+        jpype.startJVM(convertStrings=False)
+
+    # Parsing -- parsed data is stored at tempOutFile
```

Comment:
I haven't read the rest of the code yet, but is there a reason this goes to tempOutFile rather than outputFile (the argument passed in to the relevant methods just below)?

Reply:
Ah wrong comment. It is indeed outputFile. I have fixed it now.